### PR TITLE
Feautre 71 create storage for images

### PR DESCRIPTION
**Pull request by steadyGuy**

### CODE REVIEW

File: frontend/src/components/SendFileFormTest/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Box, Button, Typography } from '@mui/material';
+import React, { useState } from 'react';
+
+function SendFileFormTest() {
```
Comment (dobbi-free) [inline frontend/src/components/SendFileFormTest/index.tsx:4]: I think you should use the arrow function here

**Review by dobbi-free — CHANGES_REQUESTED** [review]
Use the arrow function where possible, to follow the same code style

**Review by dobbi-free — APPROVED** [review]

File: frontend/src/components/SendFileFormTest/index.tsx (hunk 1 above)
Comment (steadyGuy) [reply frontend/src/components/SendFileFormTest/index.tsx:4]: This is airbnb code-style, so right now, we cannot use arrow functions, only function declaration
image

Comment (steadyGuy) [reply frontend/src/components/SendFileFormTest/index.tsx:4]: And when arrow function, error appears (Function component is not a function declarationeslint[react/function-component-definition](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/function-component-definition.md)
image

**Review by Yojuks — APPROVED** [review]

File: frontend/src/components/SendFileFormTest/index.tsx (hunk 1 above)
Comment (dobbi-free) [reply frontend/src/components/SendFileFormTest/index.tsx:4]: Ok, didn't know about that

**Pull request merged**